DataFormats/Provenance/interface/LuminosityBlockAuxiliary.h

```diff
@@ -23,7 +23,11 @@ namespace edm {
                              Timestamp const& theTime,
                              Timestamp const& theEndTime)
         : processHistoryID_(), id_(theRun, theLumi), beginTime_(theTime), endTime_(theEndTime) {}
+    LuminosityBlockAuxiliary(LuminosityBlockAuxiliary&&) = default;
+    LuminosityBlockAuxiliary(LuminosityBlockAuxiliary const&) = default;
     ~LuminosityBlockAuxiliary() {}
+    LuminosityBlockAuxiliary& operator=(LuminosityBlockAuxiliary&&) = default;
+    LuminosityBlockAuxiliary& operator=(LuminosityBlockAuxiliary const&) = default;
```

Comment:
NOTE: this changed allowed a use of `std::move` to actually do something useful.